### PR TITLE
Practised with DOM Manipulation

### PR DESCRIPTION
**Pull request by azm-mehedi-hasan**

### CODE REVIEW

File: Class-3/Class Work/practise_set_1.js [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+headerElement.style.fontSize = '15px'
+
+let para = document.getElementById("paraID")
+let items = para.getElementsByClassName("button") // only changes the classes of paraID. Interesting
+for (let i = 0; i < items.length; i++) {
+    items[i].style.color = "blue"
+}
+
+let tags = document.getElementsByTagName("td");
+for (let i = 0; tags.length; i++) {
```
Comment (tajnur007) [inline Class-3/Class Work/practise_set_1.js]: I think this loop will lead us to an Infinity ;) Here we missed `i < tags.length`, right?

**Comment by azm-mehedi-hasan** [comment]
Ohhh haaa...vaii..fixing it right now

**Comment by azm-mehedi-hasan** [comment]
Vai, anything else? Any suggestion ??

**Comment by tajnur007** [comment]
Nothing else Brother. You did a great job, the practice was too good. Just keep it up.

**Comment by azm-mehedi-hasan** [comment]
OK vaia

**Review by tajnur007 — APPROVED** [review]
Great! Approved from my side :)

**Pull request merged**